app/models/conversation.js

```diff
@@ -10,5 +10,6 @@ export default Model.extend({
 
   conversationParts: hasMany('conversation-part', { async: true }),
   message: belongsTo('message', { async: true }),
-  user: belongsTo('user', { async: true })
+  user: belongsTo('user', { async: true }),
+  project: belongsTo('project', { async: true })
```

Comment:
@joshsmith since a project already exists on the `message`, I wasn't sure if there was another way to pass up the project_id to the backend than this.

Reply:
I would just make sure to alpha order here.

Reply:
I would just make sure to alpha order here.